pints/tests/test_toy_stochastic_logistic_model.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
+#
+# Tests if the stochastic logistic growth (toy) model works.
+#
+# This file is part of PINTS (https://github.com/pints-team/pints/) which is
+# released under the BSD 3-clause license. See accompanying LICENSE.md for
+# copyright notice and full license details.
+#
+import unittest
+import numpy as np
+import pints
+import pints.toy
+
+
+class TestStochasticLogistic(unittest.TestCase):
+    """
+    Tests if the stochastic logistic growth (toy) model works.
+    """
+    def test_start_with_zero(self):
+        # Set seed for random generator
+        np.random.seed(1)
+
+        # Test the special case where the initial population count is zero
+        model = pints.toy.StochasticLogisticModel(0)
+        times = [0, 1, 2, 100, 1000]
+        parameters = [0.1, 50]
+        values = model.simulate(parameters, times)
+        self.assertEqual(len(values), len(times))
+        self.assertTrue(np.all(values == np.zeros(5)))
+
+    def test_start_with_one(self):
+        # Set seed for random generator
+        np.random.seed(1)
+
+        # Run small simulation
+        model = pints.toy.StochasticLogisticModel(1)
+        times = [0, 1, 2, 100, 1000]
+        parameters = [0.1, 50]
+        values = model.simulate(parameters, times)
+        self.assertEqual(len(values), len(times))
+        self.assertEqual(values[0], 1)
+        self.assertEqual(values[-1], 50)
+        self.assertTrue(np.all(values[1:] >= values[:-1]))
+
+    def test_suggested(self):
+        np.random.seed(1)
```

Comment:
Please stick in some 1 line comments in each of the tests

Reply:
So that developers later can see e.g. what the difference between start-at-1 and simulate is

Reply:
This is not done else where...